.cirrus.yml

```diff
@@ -12,3 +12,17 @@ deploy_task:
     DEPLOY_TOKEN: ENCRYPTED[fbf2f9b7dc97cc3d960126b1540f19c36b96768c46b8eb99389e02b8c54080418d1fb90094dc2e3a36767df34acb5d5d]
   install_script: pip install pymdown-extensions
   deploy_script: ./.ci/deploy.sh
+
+markdown_tests_task:
+  container:
+    image: node:latest
+  depends_on:
+    - build
+  node_modules_cache:
+    folder: node_modules
+    populate_script: npm install -g remark-lint remark-cli remark-preset-lint-recommended && npm install -g markdown-link-check@3.7.2
+  script: 
+    - echo ========  STARTING MARKDOWN LINTING TESTS  ========
+    - find . -name \*.md -exec remark {} \;
+    - echo ========  STARTING MARKDOWN LINK CHECK TESTS  ========
+    - find . -name \*.md -exec markdown-link-check {} \;
```

Comment:
I see it reports some dead links but exits successfully. I wonder why 🤔 I checked the sources and it clearly exits with code `1` in case it detect dead links....

Reply:
It might not respect relative links. 🤔

Reply:
It sometimes is a bit fuzzy, but I think it needs links to be relative to the current file.

Reply:
Ah yes its because its testing on linux so it is looking for `/guide/` rather than `/usr/documents/cirrus-build` or wherever `$CIRRUS_WORKING_DIR` is.